File: Topic Based Problems/01 Fundamentals/Beginner/03 Sum of Two Numbers/C/SumofTwoNumbers.c
/**
 * @file SumofTwoNumbers.c
 * @author SharifdotG (sharifmdyousuf007@gmail.com)
 * @brief Sum of Two Numbers in C
 * @version 1.0
 * @date 2023-09-24
 * 
 * @copyright Copyright (c) 2023
 * 
 */

#include <stdio.h>

int main() {
    int firstNumber, secondNumber;

    printf("Enter first number: ");
    scanf("%d", &firstNumber);

    printf("Enter second number: ");
    scanf("%d", &secondNumber);

    printf("Sum of %d and %d is: %d\n", firstNumber, secondNumber, firstNumber + secondNumber);

    return 0;
}

File: Topic Based Problems/01 Fundamentals/Beginner/3 Sum of Two Numbers/Java/SumofTwoNumbers.java
/**
 * @file SumofTwoNumbers.java
 * @author SharifdotG (sharifmdyousuf007@gmail.com)
 * @brief Sum of Two Numbers in Java
 * @version 1.0
 * @date 2023-09-24
 * 
 * @copyright Copyright (c) 2023
 * 
 */

import java.util.Scanner;

public class SumofTwoNumbers {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
 
        System.out.print("Enter first number: ");
        int firstNumber = scanner.nextInt();
 
        System.out.print("Enter second number: ");
        int secondNumber = scanner.nextInt();
 
        System.out.println("Sum of " + firstNumber + " and " + secondNumber + " is: " + (firstNumber + secondNumber));
 
        scanner.close();
    }
}
 